### ISSUE TITLE
Add basic site analytics

### ISSUE THREAD

**Issue by Rabadash8820** [DerploidEntertainment/Website#6]
Google Analytics might be a little heavy here, and I'm not sure if it's free, but we should have something similar on the site, at least so we can see traffic metrics and `Referrer` values.

Resources listing the best web analytics tools out there:

- [Hotjar: Top 20 web analytics tools from our survey of 2000+ experts](https://www.hotjar.com/web-analytics/tools/)
- [First Site Guide: The top 10 most popular web analytics tools](https://firstsiteguide.com/best-website-analytics-tools/)
- [Simplilearn: 20 Best Web Analytics Tools: How to Make the Most of it?](https://www.simplilearn.com/web-analytics-tools-article)
- [Leadfeeder: 25 Best Website Analytics Tools for 2022](https://www.leadfeeder.com/blog/website-analytics-tools/)
- [Wix: 11 Best Web Analytics Tools to Boost Your Business](https://www.wix.com/blog/2020/01/best-website-analytics-tools/)
- [Elementor: 10 Leading Web Analytics Tools for 2022 [Free and Paid]](https://elementor.com/blog/best-web-analytics-tools/)
- [MonsterInsights: 9 Best Web Analytics Tools for 2022 (Free and Paid)](https://www.monsterinsights.com/best-website-analytics-tools/)
- [Blogging Wizard: 11 Best Web Analytics Tools For 2022: Get Meaningful Website Insights](https://bloggingwizard.com/web-analytics-tools/)

**Comment by Rabadash8820** [comment]
Based on the above resources, the tools with the most recommendations (in desc order) were:

1. Google Analytics (7)
2. Matomo (7)
3. Kissmetrics (7)
4. Hotjar (6)

So these are the ones I took the time to dig into on this [comparison spreadsheet](https://docs.google.com/spreadsheets/d/1-xMuHNOIUykL5B7YG43MSS6189s8cvck/edit?usp=sharing&ouid=101654406356688663182&rtpof=true&sd=true) on Google Drive. I looked at cost, .... It ultimately boils down to what we're willing to pay for, because different services are more worthwhile at different cost tiers. I can't imagine we'll ever have more than a few hundred monthly visitors, so I don't really want a recurring monthly fee for this small use case. Based on the above research the best _free tier_ service (which would be my vote) seems to be:

    TBD

@asdkiller and I will discuss and close this issue once we reach a decision.
